.release-please-manifest.json

```diff
@@ -0,0 +1,134 @@
+{
+  "src/apis/abusiveexperiencereport": "0.1.0",
+  "src/apis/acceleratedmobilepageurl": "0.1.0",
```

Comment:
I don't think we can reasonably keep a file like this up to date unless we have automation to do it.  What creates this file?  What keeps it up to date?

Reply:
I agree that this should be automated, but for the transitional step of getting things working, I'd be comfortable with us landing what we have.